stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/UiUtils.kt

```diff
@@ -40,4 +48,18 @@ internal object UiUtils {
             else -> R.drawable.stream_ic_file
         }
     }
+
+    fun getReactionTypes(): Map<String, Int> {
+        return reactionTypes
+    }
+}
+
+public enum class ReactionType(private val reactionType: String) {
```

Comment:
Will these not be customizable?

Reply:
Was it customizable before? Definitely worth making customizable. Lets do it later.

Reply:
I was asking because relying on an enum now will mean that it'll be hard to customize later on. Previously it was a `Map<String, String>`, which contained the names and emojis of the different reactions. Not sure if customization is a requirement for the new design though.

Reply:
Agree, lets postpone